site/content/docs/5.3/examples/download-app/index.html

```diff
@@ -0,0 +1,372 @@
+          </svg>
+        </svg>
+      </div>
+    </div>
+  </div>
+
+  <div class="py-4 py-md-5 bg-supporting-yellow">
+    <div class="container-xxl d-flex justify-content-center pt-md-2 pb-2">
+      <div class="col-12 col-md-6">
+        <h2 class="h1 mb-2 mb-md-3 text-center">Your feedback is valued</h2>
+        <h3 class="mb-md-3 text-center">Let us know if you spot anything to correct or improve</h3>
+        <div class="d-flex gap-md-3 flex-column flex-lg-row justify-content-between mt-4">
+          <div class="d-flex flex-column align-items-center mb-4 mb-md-0">
+            <svg xmlns="http://www.w3.org/2000/svg" class="mb-2 mb-md-4 icons-lg" width="6.25rem" height="6.25rem" role="img" focusable="false" aria-hidden="true">
+              <use xlink:href="#apple"/>
+            </svg>
+            <a href="https://github.com/Orange-OpenSource/ods-ios/issues/new/choose" class="btn btn-lg btn-secondary mt-1 mt-md-0">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-1" width="1.25rem" height="1.25rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#apple"/>
+              </svg>
+              iOS feedback
+            </a>
+          </div>
+          <div class="d-flex flex-column align-items-center">
+            <svg xmlns="http://www.w3.org/2000/svg" class="mb-2 mb-md-4 icons-lg" width="6.25rem" height="6.25rem" role="img" focusable="false" aria-hidden="true">
+              <use xlink:href="#android"/>
+            </svg>
+            <a href="https://github.com/Orange-OpenSource/ods-android/issues/new/choose" class="btn btn-lg btn-secondary mt-1 mt-md-0">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-1" width="1.25rem" height="1.25rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#android"/>
+              </svg>
+              Android feedback
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</main>
+
+<footer class="footer bg-dark navbar-dark">
+  <h2 class="visually-hidden">Sitemap & information</h2>
+  <div class="container-xxl footer-terms">
+    <ul class="navbar-nav gap-md-3">
+      <li class="fw-bold">© Orange {{< year >}}</li>
+      <li><a class="nav-link" href="https://system.design.orange.com/0c1af118d/p/0127c5-the-orange-design-system/b/135b17">Terms and conditions</a></li>
```

Comment:
There probably should be a "Cookies policy" link here too. When you load the page, accept the Mastermedia cookies, and then want to change them back to reject them, you can't without this link.

Reply:
You want something like a button to display the tarteaucitron modal ?

Reply:
Just a link like we have in the documentation in https://boosted.orange.com/docs/5.2/about/cookies/. It opens the panel directly. For example, it's done like that in https://www.orange.fr/portail (in the footer, "Gestion des cookies")